setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+from distutils.core import setup
+
+setup(
+    name='osmo_jupyter',

```

Comment:
Name should probably be inline with the repo

Reply:
proposal, after some pondering:
* change repo name to: `osmo-jupyter` ("python" and "tools" are redundant, repo names are generally hyphenated)
* change package name to `osmo_jupyter` (from python you want to `from osmo_jupyter import db_access`. unfortunately you can't use hyphen since that's a minus sign in python 🤣 )

remaining unfortunate-ness: the names don't exactly match. but it comes pretty close.

What do you think?